spec/03-plugins/14-request-transformer/02-access_spec.lua

```diff
@@ -189,6 +220,9 @@ for _, strategy in helpers.each_strategy() do
           }
         })
         assert.response(res).has.status(200)
+        local host = assert.request(res).has.header("host")
+        assert.equals(upstream_host, host)
```

Comment:
Adding this condition to _all_ subsequent (unrelated) tests really does not respect tests atomicity... If this behavior was to be broken, we'd end up with dozens of broken tests instead of one (or a few)...

Reply:
I did this because at first I tried to write an isolated regression test and I didn't hit the issue. So when I added to other tests, some presented the issue and others didn't (and it wasn't clear to me at first what the criteria was). So I ended up spreading it as an overall sanity check, a la verifying that the status remains 200.

No strong feelings about it, I'll pick one of the failing ones and make an isolated test case for it.